Tickets page second version (#1045)
EuroPython/website: src/content/pages/tickets.mdx

---
title: Sponsor
subtitle:
  Sponsoring EuroPython guarantees you highly targeted visibility
  and the opportunity to present yourself and your company to one
  of the largest and most diverse Python communities in Europe and beyond.
---

import TicketTiers from "../../components/ticket-tiers/ticket-tiers.astro";

# EuroPython 2025 Tickets
<div class="text-justify">

> Discover the latest Python trends, learn from 200+ expert speakers, network with fellow developers, enjoy social events, and explore open spaces to spark new ideas! Organized by the community for the community.

Reviewer: Only checking about the number here (e.g. if 200+ is accurate).

Reviewer 2: ~170 last year, ~192 the year before, ~150 before that:

* https://ep2024.europython.eu/speakers/
* https://ep2023.europython.eu/speakers/
* https://ep2022.europython.eu/speakers/

Author: @hugovk thanks for checking the web! Should we use some number in between, maybe `180+`? 🤔 We don't know the exact number of speakers for this year, but we have roughly the same number of slots as in the previous two years. Some events, such as summits or panel discussions, can have a varying number of speakers.

Reviewer: ΙΜΟ is safe to say 180+ based on the last year :). @hugovk thanks for sharing the info!

Author: Thanks! Updated in https://github.com/EuroPython/website/pull/1045/commits/2a306b63e71f2d9f5636047c6b93f69ca354bd70

      </li>
      <li class="flex items-start">
        <span><strong>🗓️ Saturday–Sunday, 19–20 July:</strong> Sprints Weekend</span>
      </li>
    </ul>
  </div>

## Which Ticket Tier Should I Buy?
Each ticket type is available in three pricing tiers. We trust you to choose the option that best fits your situation.

As a non-profit, every ticket you buy goes back into the Python community. Whether by our <a href="/finaid" class="text-primary hover:text-primary-hover underline">financial aid programme</a>, helping those in need of support, like students, attend the conference, or supporting Python communities across Europe through our <a href="https://www.europython-society.org/grants/" class="text-primary hover:text-primary-hover underline">EPS Grants programme</a>.

<BenefitsList>
  <BenefitItem icon="target" title="Business Tickets">
    If your company is covering your ticket or you use Python professionally, this is the right tier for you. By choosing this option, you help us keep the conference accessible. Thank you!
  </BenefitItem>
  <BenefitItem icon="network" title="Personal Tickets">

>     Are you a Python enthusiast, hobbyist, or freelancer? This ticket is for you! It’s priced for individuals paying their own way, so you can enjoy the conference at a fair and affordable rate.

Reviewer 3: ```suggestion
    Are you a Python enthusiast, hobbyist, or freelancer? This ticket is for you! It's priced for individuals paying their own way, so you can enjoy the conference at a fair and affordable rate.
```

  </BenefitItem>
  <BenefitItem icon="rocket" title="Education Tickets">
    If you're a student or an educator working in schools or universities, this ticket is for you. We offer subsidized pricing so students and educators can join in and be part of the conference!
  </BenefitItem>
</BenefitsList>

<Note>If you need a VAT invoice with your company's name, please purchase a Business Ticket. Personal Tickets do not include a company name. To assist the registration staff, your conference badge will display the ticket tier and type.</Note>

## Ticket Prices
Regular tickets are available until June 27, 2025—late-bird rates apply after.

> Combined tickets are limited to 300 due to tutorial capacity!

Reviewer 3: ```suggestion
Combined tickets are **limited to 300** due to tutorial capacity!
```
to put more emphasis

  <ul class="list-none space-y-4">
    <li class="flex items-start">
      <span>💰 Buy 5 business tickets of any type, pay for only 4</span>
    </li>
    <li class="flex items-start">
      <span>💰 Buy 10 business tickets of any type, pay for only 8</span>
    </li>
    <li class="flex items-start">
      <span>💰 Buy 15 business tickets of any type, pay for only 11</span>
    </li>
  </ul>
  </div>

<Note>Volume discounts apply only to Business Tickets. The discount is automatically applied when you add enough tickets to your basket.
  If purchasing different types of Business Tickets (e.g., Conference Only & Combined), the discount will apply to the lower-priced ticket.</Note>

#### Sponsorship
If you're purchasing multiple tickets, please consider sponsoring the event!

<div class="text-center">
  <ButtonLink url="/sponsorship/sponsor">Learn About Sponsorship</ButtonLink>
</div>

### Additional Information
      <div class="grid grid-cols-1 md:grid-cols-2 gap-10 max-w-4xl mx-auto">
        <div class="bg-white text-black p-6 rounded-lg shadow-md">
          <h5 class="text-xl font-bold mb-4">Special Grants</h5>
          <p>

>             Are you a CPython Core Developer or an EPS Fellow? You’re eligible for a free combined ticket!

Reviewer 3: ```suggestion
            Are you a CPython Core Developer or an EPS Fellow? You're eligible for a free combined ticket!
```

          </p>
          <p>
            Please check out our grant page for details on how to apply for the
            <a href="https://www.europython-society.org/core-grant/" class="text-primary hover:text-primary-hover underline">GvR grant</a> or
            <a href="https://www.europython-society.org/europython-society-fellow-grant/" class="text-primary hover:text-primary-hover underline">EuroPython Fellow grant</a>.
          </p>
        </div>

        <div class="bg-white text-black p-6 rounded-lg shadow-md">
          <h5 class="text-xl font-bold mb-4">Refund Policy</h5>
          <p class="mb-4">

>             We know plans can change! If you can’t attend, you can request a full refund via your order confirmation email until June 8, 2025.

Reviewer 3: ```suggestion
            We know plans can change! If you can't attend, you can request a full refund via your order confirmation email until June 8, 2025.
```

>             We know plans can change! If you can’t attend, you can request a full refund via your order confirmation email until June 8, 2025.

Reviewer 3: ```suggestion
            We know plans can change! If you can't attend, you can request a full refund via your order confirmation email until June 8, 2025.
```

Author: Already fixed - there were two suggestions on the same line

          </p>
          <p>
            After that, email us at <a href="mailto:refunds@europython.eu" class="text-primary hover:text-primary-hover underline">refunds@europython.eu</a> if you need a refund due to special circumstances.
            </p>
        </div>
        </div>

<br></br>

<div class="bg-white text-black p-6 rounded-lg shadow-md">
  <h5 class="text-xl font-bold mb-4">Important Notes</h5>
  <ul class="list-none space-y-4">
    <li class="flex items-start">
      <span>☕ Every ticket includes break refreshments and a light lunch for each day.</span>
    </li>
    <li class="flex items-start">
      <span>👶 Free Childcare is available at the conference for those who need it. Please make sure to register a ticket for yourself and select how many children will require childcare at checkout.</span>
    </li>
    <li class="flex items-start">
      <span>⏰ We encourage you to book your ticket early. This makes it easier for us to plan for the event and arrange important things like catering, badge printing, and other logistics.</span>
    </li>
  </ul>
  </div>

### Questions

> If you have any questions, check our <a href="/faq" class="text-primary hover:text-primary-hover underline">FAQ</a> first. Still need help? Our wonderful volunteer team at the <a href="mailto:helpdesk@europython.eu" class="text-primary hover:text-primary-hover underline">EuroPython Helpdesk</a> is happy to assist—don’t hesitate to reach out!

Reviewer 3: ```suggestion
If you have any questions, check our <a href="/faq" class="text-primary hover:text-primary-hover underline">FAQ</a> first. Still need help? Our wonderful volunteer team at the <a href="mailto:helpdesk@europython.eu" class="text-primary hover:text-primary-hover underline">EuroPython Helpdesk</a> is happy to assist—don't hesitate to reach out!
```
